bqn-mode.el

```diff
@@ -519,47 +519,56 @@ If it doesn't exist, create and return it; else, return the existing one."
       (bqn-comint-mode)
       (set-input-method "BQN-Z"))))
 
-(defun bqn-comint-process-execute-region (start end &optional dont-follow)
+(defun bqn-comint--escape (str)
+  "Targeting CBQN."
```

Comment:
A better documentation would be good, even if this is not intended to be seen by the end user.
Especially explaining why this is needed.

Reply:
I changed it to a comment instead, internal functions should generally not use docstrings.